### DISCUSSION TITLE
help identify an old acurite sensor

### DISCUSSION THREAD

**Discussion by mikestanczyk** [merbanan/rtl_433#3039] [Q&A] [answered]
I have two very old acurite sensors from a 605 weather station:

https://www.google.com/url?sa=t&source=web&rct=j&opi=89978449&url=https://www.acurite.com/media/manuals/00605-instructions.pdf%3Fsrsltid%3DAfmBOorNTBjrPv2dkcZMD3bL53XMaRgK1W0XId_YHZF77lfl5TPNO3RT

i can't find an fcc id for the sensor because one sensor is mounted in a very inaccessible place and the other is missing the battery door.

when i run rtl_433 version 23.11-143-gce56acc9 the sensors aren't recognized.  When I run with -S unknown I get packet capture files when i change batteries in the accessible sensor.

so before i go capturing a whole bunch of packets and opening a support request, can anyone can identify this sensor and if it has support?

Thanks
![IMG_20240713_132113553](https://github.com/user-attachments/assets/c37d2d75-fd07-43ba-b066-f8706819d820)
![IMG_20240713_190956401](https://github.com/user-attachments/assets/991a28ff-4afc-4b0e-af42-322ae5bab605)

**Comment by zuckschwerdt** [comment]
You can check the packets for good data by dropping them on https://triq.org/pdv/
Upload a good packet as zip here and we'll check what data is contained.

**Reply by mikestanczyk** [reply]
took me many battery insertions and removals to get a clear and distinct capture but here you are.  it decodes as 79.2 F with 20% humidity
[g001_433.92M_250k.cu8.zip](https://github.com/user-attachments/files/16742168/g001_433.92M_250k.cu8.zip)

**Reply by zuckschwerdt** [reply]
The sample looks good.
It contains a OOK PPM signal with 500 µs pulses and 500 / 1500 µs gaps and a 3000 µs packet gap. The packet is repeated 14 times with 25 bits.

I.e. this will work: `rtl_433 -X 'n=name,m=OOK_PPM,s=500,l=1500,g=2000,r=4000'`

Looks like the Acurite 590TX temperature/humidity sensor.
But the timings are somewhat off, e.g. setting `.gap_limit   = 1500,  .reset_limit = 4000,` (and maybe disabling `.sync_width`) it will show
```
model     : Acurite-590TX id        : 234
Channel   : 1            Battery   : 1             Temperature: 79.2 F       Integrity : PARITY
```

**Reply by mikestanczyk** [reply]
Ummm.  Level with me:  is this worth pursuing?  I just want a couple of temperature/humidity sensors that I can log and feed into weewx.   I already have a supported acurite weather station, an acurite 606 transmitter, and a neighbor or two with supported weather stations.  I've read that new sensors can be had for under $15 on Amazon.  (Is there a list somewhere?)  I could pull the old sensors, box them up and donate them to the community if some on was interested?

**Reply by zuckschwerdt** [reply, marked as answer]
Actually the signals you got look exactly like the ones we got at https://github.com/merbanan/rtl_433_tests/tree/master/tests/acurite/Acurite_590TX but the timings in our decoder don't really match that. I'll take a closer look and fix that. Your sensor then should just work.

I've refined the timings and docs from https://github.com/merbanan/rtl_433/pull/1411 and your sensor should now work.
